packages/oid4vci-demo-frontend/src/configs/notary.json

```diff
@@ -1,11 +1,11 @@
-{
+  {
   "general": {
-    "credentialName": "Triall Guest",
-    "issueCredentialType": "triall2023",
-    "baseUrl": "https://ssi.triall.io/demo/issuer",
-    "verifierUrl": "https://cix.triall.io",
-    "verifierUrlCaption": "Clinical Insights Exchange",
-    "downloadUrl": "https://ssi.triall.io/demo/issuer/#/download"
+    "credentialName": "Notary credential",
+    "issueCredentialType": "notary2023",
+    "baseUrl": "https://ssi.sphereon.com/demo/notary/issuer",
```

Comment:
Please look at correct urls from other PRs being worked on

Reply:
changed the general config based on @sanderPostma 's branch